test/functionalTest/cases/0000_ngsi10_subscriptions/recover_ontimeinterval_threads.DISABLED

```diff
@@ -1,348 +0,0 @@
-# Copyright 2013 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Given that OTI subscriptions will be deprecated, I have removed this file.

NTC